packer/setup.rb

```diff
@@ -187,12 +207,12 @@ def clone_or_update_ruby_by_json(h, work_dir)
   end
 end
 
-Dir.chdir("rails_ruby_bench") do
+Dir.chdir(RAILS_BENCH_DIR) do
   csystem "bash -l -c \"RAILS_ENV=profile ruby seed_db_data.rb\"", "Couldn't seed the database with profiling sample data!"
```

Comment:
@noahgibbs I was wondering, what is the reason for invoking the shell explicitly?
`-l` unfortunately resets the environment (e.g. PATH and the current ruby with `chruby`), ending up using the default ruby instead if the current one invoking the script.